boring/src/ssl/mod.rs

```diff
@@ -2553,6 +2549,15 @@ impl SslRef {
             .expect("invalid default server curves list");
     }
 
+    /// Returns an `ErrorCode` value for the most recent operation on this `SslRef`.
+    ///
+    /// This corresponds to [`SSL_get_error`].
+    ///
+    /// [`SSL_get_error`]: https://github.com/google/boringssl/blob/master/include/openssl/ssl.h#L475
+    pub fn get_error(&self, ret: c_int) -> ErrorCode {
```

Comment:
Let's call this `get_error_code`, in case we later want a `get_error` that returns a `boring::ssl::Error` like what `SslStream::make_error` produce.